lib/hibp/client.rb

```diff
@@ -114,23 +114,29 @@ def pastes(account)
     # @param password [String] -
     #   The value of the source password being searched for
     #
+    # @param add_padding [Boolean] -
+    #   Pads out the response with a random number of fake requests, to prevent
+    #   anyone looking at the responses from guessing what the hash prefix was.
+    #   
+    #
     # @note The API will respond with include the suffix of every hash beginning
     #       with the specified password prefix(five first chars of the password hash),
     #       and with a count of how many times it appears in the data set.
     #
     # @return [Hibp::Query]
     #
-    def passwords(password)
-      configure_password_query(password)
+    def passwords(password, add_padding=false)
```

Comment:
Can you pass the padding option via keyword arguments? I think it will be good for readability

Reply:
yep, good idea 😄. I'll update the PR

Reply:
Updated in 2ad96f4 😄